src/main/java/Ping/Storage.java

```diff
@@ -0,0 +1,65 @@
+package Ping;
+import java.io.*;
```

Comment:
Here, the coding standard needs imports to be stated explicitly. Perhaps you could list out all the imports, such as `java.io.File`?